domains/naufal-fifa.json

```
@@ -6,5 +6,8 @@
   },
   "records": {
     "CNAME": "cname.vercel-dns.com"
+  },
+   "txt": {
+    "_vercel": "vc-domain-verify=naufal-fifa.is-a.dev,beca9f95b34dad20b05c"
   }
```

this is not how you do it, you need to create _vercel.yourdomain.json and add the TXT records there, for more info, read our curated guide called docs (especially the step 5) -> https://docs.is-a.dev/guides/cloudflare-pages/
```suggestion
  }
```